Engines/Wine/Verbs/mfWorkaround/script.js

```diff
@@ -0,0 +1,113 @@
+const Wine = include("engines.wine.engine.object");
+const Resource = include("utils.functions.net.resource");
+const { Extractor } = include("utils.functions.filesystem.extract");
+const { ls, cp } = include("utils.functions.filesystem.files");
+//const { getGithubReleases } = include("utils.functions.net.githubreleases"); commented out because it's useless until z0z0z will use releases
+
+const Optional = Java.type("java.util.Optional");
+const Regedit = include("engines.wine.plugins.regedit");
+const Regsvr32 = include("engines.wine.plugins.regsvr32");
+const OverrideDLL = include("engines.wine.plugins.override_dll");
+
+/**
+ * Verb to install Media Foundation workaround
+ * see: https://github.com/z0z0z/mf-install
+ */
+class mfWorkaround {
+    constructor(wine) {
+        this.wine = wine;
+    }
+
+    /**
+     * Sets the used mfWorkaround version
+     *
+     * @param {string} mfWorkaroundVersion The version of mfWorkaround to downlaod
+     * @returns {mfWorkaround} The mfWorkaround object
+     */
+    withVersion(mfWorkaroundVersion) {
+        this.mfWorkaroundVersion = mfWorkaroundVersion;
+
+        return this;
+    }
+
+    go() {
+        const wizard = this.wine.wizard();
+        const prefixDirectory = this.wine.prefixDirectory();
+        const system32directory = this.wine.system32directory();
+        const system64directory = this.wine.system64directory();
+
+        //commented out because it's useless until z0z0z will use releases
+        /*if (typeof this.mfWorkaroundVersion !== "string") {
+            const versions = getGithubReleases("Kron4ek", "FAudio-Builds", wizard);
+            this.mfWorkaroundVersion = versions[0];
+        }*/
```

Comment:
The same applies to these comments

Reply:
I've left them because someone may notice gthub link and ask why we are not using `getGithubReleases` function.

Reply:
@plata what do you think should we retain these comments or remove them?

Reply:
Remove. If @Zemogiter would like to have a comment, use something like:
```
// TODO: use GitHub releases if repository provides releases
```
We should not keep commented code like this. It will likely get outdated and only pollutes the script.